### DISCUSSION TITLE
Change dir

### DISCUSSION THREAD

**Discussion by theun78** [hassio-addons/addon-ftp#214] [Q&A] [answered]
I want to save files instead of the "share" dir, on a USB HDD, how can I change the storage path?

Thanks

**Comment by frenck** [comment, marked as answer]
That is a question for Home Assistant, not this add-on.

../Frenck

**Reply by theun78** [reply]
why do you say it's a question of home assistant?
I need only ftp files to be saved in a folder other than the default one. Usually add-ons give the possibility to choose which folder to save in

**Comment by frenck** [comment]
> on a USB HDD

Because using an USB HDD in HA is not possible atm. Thus not an add-on problem.
